### ISSUE TITLE
<08/09/23><NYU><Android><Talkback><Purple Rides><waiting timer is not visible at user end> <Parent_Id-1980>

### ISSUE THREAD

**Issue by krupal-14** [nammayatri/nammayatri#3089]
#1980 
Waiting timer is not displaying at user end.

Steps To reproduce
1.launch the apk.
2.Login with valid credentials.
3.Book a ride.
4.click on i have arrived from driver end and observe.

Expected result
User should get waiting timer

Actual result
User is not getting the waiting timer.

**Comment by MercyQueen** [comment]
It is showing the wait timer
